src/etcpal/netint.c

```diff
@@ -238,3 +221,22 @@ int compare_netints(const void* a, const void* b)
 
   return (netint1->index > netint2->index) - (netint1->index < netint2->index);
 }
+
+/*!
+ * \brief Refresh the list of network interfaces.
+ *
+ * Rebuilds the cached array of network interfaces that is returned via the
+ * etcpal_netint_get_interfaces() function. If the refresh operation results in a different list
+ * (there is a different number of network interfaces, or any interface has changed IP settings),
+ * *list_changed is set to true.
+ *
+ * \param[out] list_changed Set to true if the set of interfaces has changed in any way.
+ * \return #kEtcPalErrOk: Interfaces refreshed.
+ * \return Other error codes from the underlying platform are possible here.
+ */
+etcpal_error_t etcpal_netint_refresh_interfaces(bool* list_changed)
+{
+  // TODO
```

Comment:
I'm assuming there's an Issue/SCR for this?

Reply:
https://jira.etcconnect.com/browse/ETCPAL-20